recipes/dsfmt/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "dSFMT" %}
+{% set version = "2.2.3" %}
+{% set sha256 = "82344874522f363bf93c960044b0a6b87b651c9565b6312cf8719bb8e4c26a0e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.math.sci.hiroshima-u.ac.jp/~m-mat/MT/SFMT/{{ name }}-src-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: build.sh
+
+requirements:
+  build:
+    - gcc
```

Comment:
Would change this to `toolchain`. We should be able to build this with the system included compilers I think.